content/en/software_catalog/manage.md

```diff
@@ -41,7 +41,7 @@ algolia:
 
 ## Assigning an owner 
 
-You can assign a `team` to entries in the Software Catalog either in the UI or by creating a [Service Definition][4]. Datadog recommends that you set up [Datadog Teams][5] so that you can specify individual members of the team and take advantage of *Teams* filters across common views like Dashboards and Notebook lists. 
+You can assign a `team` to entries in the Software Catalog either in the UI or by creating a [Service Definition][4]. Datadog recommends that you set up [Datadog Teams][5] so that you can specify individual members of the team and take advantage of *Teams* filters across common views like Dashboards and Notebook lists. The ability ot sync Datadog Teams with your identify providers (IdPs) is in [Preview][11].
```

Comment:
```suggestion
You can assign a `team` to entries in the Software Catalog either in the UI or by creating a [Service Definition][4]. Datadog recommends that you set up [Datadog Teams][5] so that you can specify individual members of the team and take advantage of *Teams* filters across common views like Dashboards and Notebook lists. The ability to sync Datadog Teams with your identity providers (IdPs) is in [Preview][11].
```